src/odoc/bin/main.ml

```diff
@@ -45,19 +45,19 @@ let odoc_file_directories =
     "Where to look for required .odoc files. \
      (Can be present several times)."
   in
-  Arg.(value & opt_all convert_directory [] @@
+  Arg.(value & opt_all convert_directory [] &
```

Comment:
Is this the `&` operator from pervasives? If so, it's now supposed to be `&&`...

Reply:
No. That's the [function application defined in Cmdliner](http://erratique.ch/software/cmdliner/doc/Cmdliner.Arg.html#VAL(&)). It's equivalent to [`(@@)` in Pervasives](https://docs.mirage.io/ocaml/Pervasives/index.html#val-(@@)). AFAIK Cmdliner introduced the `&` operator before `@@` was added to Stdlib.

The reason I changed `@@` to `&` was because the current usage is inconsistent and most people use `&` with Cmdliner.

Reply:
So, you can see my confusion. Perhaps we should encourage everyone to go into the other direction given that it really means `@@`?

Reply:
I agree. We should move away from operators that duplicate the functionality of later standard operators. I think `&` should probably be deprecated in Cmdliner.

Reply:
@pmetzger @bluddy while I tend to agree with you, in this case prefer following the current established convention. Both `&` and `@@` are mixed in current code which is more confusing than either of them exclusively IMO.

Feel free to open an issue for this on the Cmdliner repo (CC @dbuenzli).

Reply:
We can handle it by moving everything to `@@` surely, yes?

Reply:
You'll just confuse everyone who's used to cmdliner (which is most OCaml programmers). It's DSL is well established, and most people just know that you put `&` there and don't necessarily know that it is apply.

Reply:
Gotcha.